docs/source/modules/mixedlayer.rst

```diff
@@ -0,0 +1,277 @@
+Mixed layer module
+=====================
+
+Summary
+----------------------
+This module updates the sea surface temperature (SST, :math:`T_s`). 
+
+SST boundary condition options
+-----------------------
+The SST options are:
+    - an input SST file (``do_sc_sst`` namelist option)
+    - a slab ocean aquaplanet analytic form (``prescribe_initial_dist`` namelist option)
```

Comment:
Can you add a reference here please? I assume one of Frierson's papers but check the form in the paper to make sure.

Reply:
Sorry don't put the ref here but in the section below.